### PR TITLE
Adding support for McAfee Endpoint Security

### PR DESCRIPTION
**Pull request by jvanlieshout**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,57 @@
...
 For the agent you'll need the FramePkg.exe and for the virusscan the av.zip.
 Both files should be placed on your local repo and defined in the attributes.
 
+The cookbook now also supports installation of McAfee Endpoint Security.
+In order to do so ovrewrite the virusscan attributes like this
+
+```ruby
+overwrite['mcafee']['virusscan']['package_name']      = 'McAfee Endpoint Security Platform'
+overwrite['mcafee']['virusscan']['options']           = 'ADDLOCAL="tp" /qn /nocontentupdate'
+overwrite['mcafee']['virusscan']['installer_exe']     = 'setupEP.exe'
```
Comment (svanharmelen) [inline README.md]: There is no need to override them. Just setting them is enough.

Additionally it's `override`, not `overwrite` :wink:

File: attributes/default.rb [hunk 2]
```diff
@@ -17,13 +17,21 @@
...
+default['mcafee']['virusscan']['options']             = 'ADDLOCAL=ALL RUNAUTOUPDATESILENTLY=True REMOVE=ScriptScan,LotusNotesScan,EmailScan REBOOT=R /qn'
+default['mcafee']['virusscan']['installer_timeout']   = 900
+default['mcafee']['virusscan']['installer_exe']       = 'SetupVSE.exe'
+
+# If you want to install the newer McAfee Endpoint Security overwrite these virusscan attributes like this:
+# overwrite['mcafee']['virusscan']['package_name']      = 'McAfee Endpoint Security Platform'
+# overwrite['mcafee']['virusscan']['url']               = <url to the zip file>
+# overwrite['mcafee']['virusscan']['checksum']          = <corresponding checksum>
+# overwrite['mcafee']['virusscan']['options']           = 'ADDLOCAL="tp" /qn /nocontentupdate'
+# overwrite['mcafee']['virusscan']['installer_exe']     = 'setupEP.exe'
```
Comment (svanharmelen) [inline attributes/default.rb]: Same here... Don't override them, just set them to the other values...

**Comment by jvanlieshout** [comment]
Allright, fixed it! Thanks

File: README.md (hunk 1 above)
Comment (jvanlieshout) [reply README.md]: fixed

File: attributes/default.rb (hunk 2 above)
Comment (jvanlieshout) [reply attributes/default.rb]: also fixed

**Pull request merged**